robot_id : "robot_id"

###log_save_falg
battery_log_save_flag : True
air_log_save_flag : True


###pose_setting
start_pose : {                      #home. 로봇 시동할 시에 좌표점
    position_x : -1.89478651836,
    position_y : 1.95743074852,
    orientation_z : -0.00623354600761,
    orientation_w : 0.999980571263
    # position_x : -0.0224158604738, # first start pose (in kiria office)
    # position_y : -1.26700368463,
    # orientation_z : 0.687481585026,
    # orientation_w : 0.726201810966
}
charging_station_pose : {           #충전스테이션 도킹 진입시 좌표점
    position_x : -1.78062586667,
    position_y : 2.00167680295,
    degree : -14.7372075181
    #2nd floor test
    # position_x : 0.826682692089,
    # position_y : -0.892144786376,
    # degree : 343
    # dgcity_Confio /AMCL_particles

    # position_x : 0.231736971407,
    # position_y : -1.95627267241,
    # degree : 330
    # dgcity_Hallow
    # position_x : -0.117557688382,
    # position_y : -0.0947024776856,
    # degree : 20
}
docked_pose : {
    # dgcity_Conf
    position_x : 1.01132829772,
    position_y : -0.884276888525,
    orientation_z : -0.0791007454498,
    orientation_w : 0.996866627022
    # dgcity_Hallow
    # position_x : -1.30682382235,
    # position_y : -0.105334295375,
    # orientation_z : -0.00409012170155,
    # orientation_w : 0.999991635417
}
# robot_init_pose : {                 #도킹 진입시 로컬라이제이션 조정 좌표(dgcity_Hallow map)
#     # dgcity_Conf
#     # position_x : 0.231736971407,  
#     # position_y : -1.95627267241,
#     # orientation_z : 0.0710041607353,
#     # orientation_w : 0.99747601934,
#     # dgcity_Hallow
#     # position_x : -0.879012809112,
#     # position_y : -0.116086582339,
#     # orientation_z : -0.0032502662301,
#     # orientation_w : 0.999994717871
# }

robot_init_pose : {                 #도킹 진입시 로컬라이제이션 조정 좌표(dgcity_sp map)
    # position_x : -2.4868389948,
    # position_y : -1.41513257093,
    # orientation_z : -0.0107816950431,
    # orientation_w : 0.999941875837
    position_x : -1.77258133539,
    position_y : 2.00849793116,
    orientation_z : -0.0230088983036,
    orientation_w : 0.999735260256
}

server_url : "http://192.168.0.51:8000/file/upload/"